src/components/header/Header.js

```diff
@@ -0,0 +1,16 @@
+import Navbar from './navigationBar';
+
+const Header = () => {
+    return (
+        <header>
+            <div className="nav-area">
+                <a href="/Users/Іра/IdeaProjects/project/salon/src/pages" className="logo">
```

Comment:
here Link should be used and to param of the link should be '/' for now as no pages currently